src/typeorm/typeorm.config.ts

```diff
@@ -23,15 +28,16 @@ export const typeOrmAsyncConfig: TypeOrmModuleAsyncOptions = {
   },
 };
 
-export const typeOrmConfig: TypeOrmModuleOptions = {
-  type: 'mysql',
-  host: process.env.DB_HOST,
-  port: parseInt(process.env.DB_PORT, 10),
-  username: process.env.DB_USERNAME,
-  database: process.env.DB_NAME,
-  password: process.env.DB_PASSWORD,
-  autoLoadEntities: true,
+export const typeOrmConfig: DataSourceOptions = {
+  type: 'postgres',
+  host: 'localhost',
+  port: 5432,
+  database: 'open_forum_api',
```

Comment:
In the set-up you are creating a database called postgres. So I would not expect this to work. Also isn't there an environment variable for this? Why not use it in both places?